code/go/0chain.net/smartcontract/faucetsc/models.go

```diff
@@ -113,6 +114,16 @@ func (gn *GlobalNode) updateConfig(fields map[string]string) error {
 			gn.OwnerId = value
 
 		default:
+			if strings.HasPrefix(key, Settings[Cost]) {
+				costKey := strings.TrimPrefix(key, Settings[Cost]+".")
+				costValue, err := strconv.Atoi(value)
+				if err != nil {
+					return fmt.Errorf("key %s, unable to convert %v to integer", key, value)
+				}
+				gn.Cost[costKey] = costValue
```

Comment:
Even the validateCostConfig will filter it out, we should still check if `costKey` is in the `costFunctions` to  return error early here.

Reply:
sure